modules/ide/ide-migration/src/main/resources/META-INF/dirigible/ide-migration/services/migration-launch-view.js

```diff
@@ -9,14 +9,15 @@
  * SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and XSK contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-exports.getView = function () {
-	var view = {
-		"id": "migration-launch",
-		"name": "Migration Launch",
-		"factory": "frame",
-		"region": "main",
-		"label": "Migration Launch",
-		"link": "../ide-migration/migration-launch.html"
-	};
-	return view;
-}
\ No newline at end of file
+package com.sap.xsk.ide.migration;
+
+import static org.junit.Assert.assertTrue;
+
+import org.graalvm.polyglot.HostAccess;
+
+public class AssertHolder {
+  @HostAccess.Export
+  public void callAssertTrue(Boolean res) {
+      assertTrue(res);
+  }
+}
```

Comment:
I really don't get this change

Reply:
Maybe try to build the module/whole project with:
- `mvn clean install -P content` - to fetch the content from the xsk-ide-migration project in DirigibleLabs
- `mvn clean install` - to update the license headers